keg_elements/forms/__init__.py

```diff
@@ -200,6 +201,26 @@ def selected_choice_label(self):
         return value_dict.get(self.data)
 
 
+class RequiredRadioBooleanField(wtforms.fields.RadioField):
+    def __init__(self, *args, **kwargs):
+        yes_label = kwargs.pop('yes_label', 'Yes')
+        no_label = kwargs.pop('no_label', 'No')
```

Comment:
Yes/no labels should be "true_label" and "false_label".  This is a boolean field and internally, we should be dealing with true/false and have default labels of "yes"/"no".